tests/simpleKVBC/TesterReplica/internalCommandsHandler.cpp

```diff
@@ -263,6 +316,23 @@ std::optional<std::map<std::string, std::string>> InternalCommandsHandler::getBl
     }
   }
 
+  // Handle main key updates which should be ignored by the tests
```

Comment:
`  // Handle main key updates which should be ignored by the tests` ->
`   // Handle main key updates which should be ignored by the linzeriability checker of apollo tests` (or something similar).
It is ignored only by the skvbc tracker, right?